lib/xcpretty/formatters/t23Formatter.rb

```diff
@@ -0,0 +1,99 @@
+module XCPretty
+    class T23Formatter < Simple
+        def format_t23_output(text)
+            INDENT + INDENT + text[4..-1] #Ignore the T23: prefix
```

Comment:
This 4 is driven by the length of the prefix (also seen inside T23_MATCHER below). 

Can you pull the prefix into a constant and share it between them?